base-notebook/Dockerfile.ppc64le

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) IBM Corporation 2016 
```

Comment:
Thanks, sorry for the delay. I was waiting for Travis' green check, but had to get on a plane. Thanks for your patience!